service/src/test/resources/ehr/request/fhir/output/expected-xml-with-effective-time.xml

```diff
@@ -27,7 +27,7 @@
             <id root="test-id-2" />
             <statusCode code="COMPLETE" />
             <effectiveTime>
-                <low value="20100113"/><high value="20130713162000"/>
+                <low value="20100113"/>
```

Comment:
High value disappeared because the corresponding ehrComposition doesn't exist in this output anymore.